pkg/master/controller.go

```diff
@@ -104,14 +105,24 @@ func (c *completedConfig) NewBootstrapController(legacyRESTStorage corerest.Lega
 		ExtraEndpointPorts:        c.ExtraConfig.ExtraEndpointPorts,
 		PublicServicePort:         c.GenericConfig.ReadWritePort,
 		KubernetesServiceNodePort: c.ExtraConfig.KubernetesServiceNodePort,
+		controllerName:            "bootstrap-controller",
 	}
 }
 
+func (c *Controller) Name() string {
+	return c.controllerName
```

Comment:
just meant a simple constant in the package. This func does not add anything to that.